### PR TITLE
Release 2.5.1

### PR DESCRIPTION
**Pull request by pwtyler**
## CHANGELOG
* Fix Solr not indexing automatically [[#598](https://github.com/pantheon-systems/solr-power/pull/598)]
* Fixed issue with empty/zero value for posts_per_page [[#599](https://github.com/pantheon-systems/solr-power/pull/599)] (props @threeiem)
* Testing and automation improvements

### CODE REVIEW

**Comment by pwtyler** [comment]
Flipping to draft as I'd like to investigate a potential issue with `commit` cli command introduced in 2.4.0 before cutting a release.

**Comment by pwtyler** [comment]
Alright, not going to block on `wp solr commit`, will dig into that another day.

File: package.json [hunk 1]
```diff
@@ -23,4 +23,4 @@
     "grunt-wp-readme-to-markdown": "~2.1.0",
     "node-sass": "^9.0.0"
   }
-}
+}
```
Comment (rwagner00) [inline package.json:26]: does a json file need to end in a black line?

**Review by rwagner00 — APPROVED** [review]
As long as the JSON end line question is fine, LGTM

File: package.json (hunk 1 above)
Comment (pwtyler) [reply package.json:26]: npm won't care here.

File: includes/class-solrpower-wp-query.php [hunk 2]
```diff
@@ -223,27 +228,28 @@ function posts_request( $request, $query ) {
...
 		}
 
-		$this->highlighting = $search->getHighlighting();
-
-		$search = $search->getData();
-
-		$search_header      = $search['responseHeader'];
-		$search             = $search['response'];
-		$query->found_posts = $search['numFound'];
+		$this->highlighting                               = $search->getHighlighting();
```
Comment (jazzsequence) [inline includes/class-solrpower-wp-query.php:246]: I hate ridiculous spacing changesets like these. It's why I've disabled the sniff to align variable and array values in our coding standards. In the future, I'd be in favor of deleting the spaces between things and not aligning anything. It's a waste of development time tapping the spacebar a bunch of times. 🤬

**Pull request merged**